arrow-core-data/src/main/kotlin/arrow/core/Const.kt

```diff
@@ -15,33 +15,43 @@ fun <A, T> ConstOf<A, T>.value(): A = this.fix().value()
 data class Const<A, out T>(private val value: A) : ConstOf<A, T> {
 
   @Suppress("UNCHECKED_CAST")
-  fun <U> retag(): Const<A, U> = this as Const<A, U>
+  fun <U> retag(): Const<A, U> =
+    this as Const<A, U>
 
   @Suppress("UNUSED_PARAMETER")
-  fun <G, U> traverse(GA: Applicative<G>, f: (T) -> Kind<G, U>): Kind<G, Const<A, U>> = GA.just(retag())
+  fun <G, U> traverse(GA: Applicative<G>, f: (T) -> Kind<G, U>): Kind<G, Const<A, U>> =
+    GA.just(retag())
 
   @Suppress("UNUSED_PARAMETER")
-  fun <G, U> traverseFilter(GA: Applicative<G>, f: (T) -> Kind<G, Option<U>>): Kind<G, Const<A, U>> = GA.just(retag())
+  fun <G, U> traverseFilter(GA: Applicative<G>, f: (T) -> Kind<G, Option<U>>): Kind<G, Const<A, U>> =
+    GA.just(retag())
 
   companion object {
-    fun <A, T> just(a: A): Const<A, T> = Const(a)
+    fun <A, T> just(a: A): Const<A, T> =
+      Const(a)
   }
 
-  fun value(): A = value
+  fun value(): A =
```

Comment:
I'm wondering out loud, wouldn't be simpler to just expose `value` in the first place instead?

Reply:
This was originally done to expose a `value()` method on both the concrete and kinded `xxxOf` variants.

Perf-wise this doesn't cause any impact one way or the other.